### PR TITLE
Feature to add login and registration pages 

### PR DESCRIPTION
**Pull request by NeemaPeter**
The PR adds the feature to register a new user and login with the same user id and password using the spring security.
When the user enters invalid credentials we get a handled Invalid message.
Screenshot 2023-06-06 at 6 38 13 AM
Screenshot 2023-06-06 at 6 38 26 AM
Screenshot 2023-06-06 at 6 48 07 AM
Screenshot 2023-06-06 at 6 50 13 AM

### CODE REVIEW

**Comment by VrushabhPatravali** [comment]
Looks good!

**Review by VrushabhPatravali — APPROVED** [review]

File: src/.DS_Store
Comment (Surinder07) [inline src/.DS_Store:1]: What is this .DS_Store file? could you remove this

**Review by Surinder07 — APPROVED** [review]

File: src/.DS_Store
Comment (NeemaPeter) [reply src/.DS_Store:1]: It was a Mac generated file, I have removed it now

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1 +1,7 @@
 
+spring.datasource.url=jdbc:mysql://localhost:3306/login_system
+spring.datasource.username=root
+spring.datasource.password=User@123
+
```
Comment (Surinder07) [inline src/main/resources/application.properties:5]: remove this extra space. Ignore (only test comment)

File: src/main/resources/.DS_Store
Comment (Surinder07) [inline src/main/resources/.DS_Store:1]: please remove DS_store or add to git ignore

**Review by dilpreetbhogal — APPROVED** [review]

**Review by Surinder07 — APPROVED** [review]
Look fine to me. Approved.

**Pull request merged**